texts/epn_paper.md

```diff
@@ -0,0 +1,74 @@
+Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
+With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.
+They are also more inclusive and reach a broader audience especially if the talks are recorded.
+
+![](planning_checklist.png)
+
+**Figure caption:** A conference organizer checklist.
+
+<!-- #### We decided to develop online conferences and established VSF. -->
+
+The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.
+We started exploring the opportunities and challenges of online conferences in late Summer 2019—well before the COVID-19 pandemic.
+We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
+It quickly became clear that we had the capacity to establish a venue for virtual conferencing. 
+So, we committed ourselves to developing this idea into a real platform, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).
+
+<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
+
+As we started to organize ourselves, we realized right away that in order to achieve our goals, we needed to revisit our assumptions about conferences.
+For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
+The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions.
+Likewise, the length of an online event is constrained by very different factors, such as the spread of timezones, local commitments and distractions of the participants.
+
+<!-- #### We ran a couple of events, and saw that they work well. -->
+
+We created toolkits and developed strategies and guidelines to navigate these challenges.
+And we got to work to organize online events, starting with a beta test workshop in December 2019.
+We learned in real time about challenges of coordinating live questions, addressing technical glitches, and giving feedback to the audience and the speaker.
+More importantly, however, we were impressed by the quality of the event and the possibility of creative ways of engagement in an online setting.
+For example, "watching parties" with a number of participants attending as a group was effective in producing local discussions that were then shared online. And one of our speakers guided the audience through the technical parts of their talk using live code.
```

Comment:
```suggestion
Some of our attendees planned their own "watching parties" with a number of participants viewing as a group that pre-discussed their questions and afterwards shared them with the speaker.
Overall we received overwhelmingly positive feedback from all participants.
```

An attentive reader will notice that also conventional presentations easily include live coding.